README.md

```diff
@@ -1,31 +1,47 @@
 # noir-ethereum-history-api
 
-_noir-ethereum-history-api_ is a [Noir](https://noir-lang.org) library for proving and verifying account balances and smart contracts storage slots on the Ethereum blockchain and other EVMs.
+**noir-ethereum-history-api** is a [Noir](https://noir-lang.org) library for proving and verifying historical data on the Ethereum blockchain and other EVM compatible blockchains.
 
-As for now, it allows for proving and verifying last 256 blocks.
+It allows you to prove:
 
-It is currently in a development stage.
+- Account state
+  - `balance`, `nonce`, `codeHash`, `storageRoot`
+  - Example: _`Vitalik's` balance was `5 ETH` at block `N`_
+- Storage values by key
+  - Example: _`Uniswap` contract had value `5` at storage slot `0` at block `N`_
+- Transaction inclusion
+  - Example: _Transaction number `115` in block `N` was sending `3 ETH` to address `to`_
+- Receipt inclusion
+  - Example: _Transaction number `115` in block `N` succeeded (`status == 1`) and used `100k` gas_
+- Log inclusion
+  - Example: _Log number `0` within transaction number `115` in block `N` has `topic0 === 0x...`_
+
+As of now, it allows for proving and verifying last 256 blocks.
```

Comment:
I would more go like sth like that:
Additionally, we provide [smart contracts](link) that allow to verify inclusion of block in the chain for last 256 blocks.

For example in Aztec (I think) you can work with all past block hashes.